### PR TITLE
Adding charts

### PR DESCRIPTION
**Pull request by akstarre**
Added CoinCharts and Sparkline Charts to app. Minor styling changes. Still need to go back and add headers and other misc styles.

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -1,10 +1,9 @@
 "use client";
 
 import { Navbar } from "@/components/Navbar";
-import { Chart } from "@/components/Chart";
 import MarketTable from "../components/MarketTable/index";
 import { CoinSelectorCarousel } from "../components/CoinSelectorCarousel/index";
-import { CoinPortfolioSwitch } from "@/components/CoinPortfolioSwitch";
+import { CoinCharts } from "@/components/CoinCharts";
```
Comment (yoloOnTheBattlefield) [inline src/app/page.tsx]: move it be fore the relative imports

Comment (yoloOnTheBattlefield) [reply src/app/page.tsx]: imports priority:
node_modules 
absolute imports made by you
relative imports

File: src/components/MarketListItem/index.tsx [hunk 2]
```diff
@@ -9,6 +9,8 @@ import { getCaretAndColor } from "@/utils/formatting";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import Image from "next/image";
 import tw from "tailwind-styled-components";
+import { SparklineChart, CoinDataProps } from "../SparklineChart";
+import { transformSparklineToChartFormat } from "@/utils/conversions";
 
 // type PercentChangeProp = {
 //   color: string;
```
Comment (yoloOnTheBattlefield) [inline src/components/MarketListItem/index.tsx:16]: no no

File: src/components/SparklineChart/index.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+  };
+
+  const data = {
+    labels: reducePoints(coinData.prices, 4).map((price, i) => {
+      let hour = new Date(price[0]).getHours();
+      const amPm = hour >= 12 ? "PM" : "AM";
+      hour = hour % 12;
+      hour = hour || 12;
+      return (hour < 10 ? "0" : "") + hour;
+    }),
```
Comment (yoloOnTheBattlefield) [inline src/components/SparklineChart/index.tsx]: the other function above does the same thing?

**Review by yoloOnTheBattlefield — CHANGES_REQUESTED** [review]

File: src/components/SparklineChart/index.tsx (hunk 3 above)
Comment (akstarre) [reply src/components/SparklineChart/index.tsx]: reducePoints skips every so many points of data to reduce the points shown on a graph. Return hour just formats the axis. It's actually not needed in sparkline so I'll remove/modify it.

**Review by yoloOnTheBattlefield — APPROVED** [review]

**Pull request merged**